tasks/components/Avatar.tsx

```diff
@@ -1,20 +1,30 @@
-import { tw, tx } from '@helpwave/common/twind/index'
+import { tw, tx, css } from '@helpwave/common/twind/index'
 
 export type AvatarProps = {
   avatarUrl: string,
   alt: string,
   size?: 'tiny' | 'small' | 'medium' | 'large'
 }
 
+const avatarStyle = css`
+.alt-text {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+`
```

Comment:
Would be nice to have this inline as twind too.
`flex justify-evenly items-center`
It's way more readable.